hello-world/app/js-vanilla/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "templates-app-vanilla-javascript",
+  "description": "Polywrap dApp Vanilla Javascript Template",
+  "private": true,
+  "version": "0.0.1-prealpha.1",
+  "scripts": {
+    "start": "serve public",
+    "build": "yarn codegen && rollup -c",
+    "codegen": "npx w3 app codegen"
+  },
+  "dependencies": {
+    "@web3api/client-js": "latest",
```

Comment:
Currently demo projects use a fixed version of the toolchain. This is so that they can always consistently run for developers who want to build / run the demo locally. I think we should continue to do this, and build some sort of automatic upgrade process that can run when new releases of the toolchain are created. That way we can see what breaks and fix it before it hits the master branch.